scalableminds/webknossos
# Send Analytics Event for Failed Dataset Upload

Track failed dataset uploads.
Sorry for the large diff in formatting, i did not change the `.then` block in dataset_upload_view, only added the catch before.

### Steps to test:
- insert `          _ <- Fox.failure("nope")` in `app/com/scalableminds/webknossos/datastore/controllers/DataSourceController.scala` Line 130
- in `application.conf`, set `backendAnalytics.verboseLoggingEnabled = true`
- Upload a dataset
- request should fail
- backend logging should show analytics event for failed request

### Issues:
- fixes #5168 

------
